### PR TITLE
<3 units

### PR DESCRIPTION
**Pull request by rsheeter**
Because I confused myself :)

Hopefully helps with #22.

[Rendered](https://github.com/googlefonts/fontmake-rs/blob/units/resources/text/units.md)

### CODE REVIEW

File: resources/text/units.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+Written because the author confused themselves and wanted to be sure they
+now understand.
+
+There are three main classes of coordinates in fonts:
+
+1. Design coordinates
+   * Used by the type designer and design tool
+   * Not directly present in the font binary at all
+   * For example, [<dimension xvalue/>](https://fonttools.readthedocs.io/en/latest/designspaceLib/xml.html#dimension) in a .designspace file
+1. User, aka External, coordainates
```
Comment (cmyr) [inline resources/text/units.md:12]:
```suggestion
1. User, aka External, coordinates
```

File: resources/text/units.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+   * The values in [fvar](https://learn.microsoft.com/en-us/typography/opentype/spec/fvar), for example Weight 400
+   * What you would expose to an end user in a document editor or manipulate via CSS
```
Comment (cmyr) [inline resources/text/units.md:14]: the use of a numerical value to denote weight feels more like some ancient convention, and I don't really think of this in terms of 'coordinates', per se, but I can see how that makes sense. I'm now very curious to think about where this convention came from...

**Review by cmyr — APPROVED** [review]
Reading the introduction here I was getting excited about a deep dive on picos, points, and ems (oh my)

File: resources/text/units.md (hunk 2 above)
Comment (rsheeter) [reply resources/text/units.md:14]: @davelab6 do you happen to know where the weight scale comes from?

File: resources/text/units.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+# Units
```
Comment (anthrotype) [inline resources/text/units.md:1]: I think this discussion is not about font "units" or "coordinates" in general, but more specifically about _variation axis_ coordinates.

File: resources/text/units.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+Written because the author confused themselves and wanted to be sure they
+now understand.
+
+There are three main classes of coordinates in fonts:
```
Comment (anthrotype) [inline resources/text/units.md:6]: ``diff
- There are three main classes of coordinates in fonts:
+ There are three main classes of axis coordinates in variable fonts:
```

Comment (rsheeter) [reply resources/text/units.md:6]: Will apply that edit directly as the merge button has gone away

Comment (rsheeter) [reply resources/text/units.md:6]: Doen in https://github.com/googlefonts/fontmake-rs/commit/67b44d2b88e0daaac9680c5f40df0d062d0676f0

**Pull request merged**